exporter/pagerdutyexporter/README.md

```diff
@@ -0,0 +1,53 @@
+# PD Exporter
+
+Exports data to a PD back-end.
```

Comment:
Please use the full company name when writing text to help with users adopting it. (remove as much confusion as possible)

```suggestion
Exports data to Pager Duty backend
```